doc/holo-files.8.pod

```diff
@@ -98,6 +98,14 @@ This means that when configuration is applied as a holoscript on top of the
 default configuration, future updates to the default configuration will
 automatically be picked up, as long as C<holo apply> is run after the system update.
 
+When detecting these files, to know which suffixes to look for, holo-files
+inspects C<ID> and C<ID_LIKE> in L<os-release(5)> to which family the operating
+system belongs to, and thus which package manager is used.  It currently
+recognizes C<arch>, C<debian>, C<fedora>, C<suse>, and (for testing purposes)
+C<unittest>.  If for some reason L<os-release(5)> can't be relied upon, then it
+may be overridden by setting the C<$HOLO_FILES_CURRENT_DISTRIBUTION> environment
+variable to one of the recognized distributions.
```

Comment:
Please keep `unittest` and `$HOLO_FILES_CURRENT_DISTRIBUTION` out of the official documentation. Both are strictly testing instruments. If `os-release(5)` cannot be relied upon, users should be pushed into submitting bug reports to their distribution (or to us) instead of monkeying around with their environment.